examples/vue/.storybook/main.ts

```diff
@@ -16,7 +16,10 @@ export default {
   async viteFinal(config, { configType }) {
     // Demonstrates use of mergeConfig and resolve.alias as an array
     return mergeConfig(config, {
-      resolve: { alias: [{ find: '@assets', replacement: resolve(__dirname, '..', 'stories', 'assets') }] },
+      resolve: {
+        alias: [{ find: '@assets', replacement: resolve(__dirname, '..', 'stories', 'assets') }],
+        dedupe: ['@storybook/client-api'],
```

Comment:
This `dedupe` is only necessary if there are different storybook versions being installed. I see that `@storybook/addon-interactions` doesn't match the rest, but from discord it sounded like this error started before you added that addon.  If you install `addon-interactions` 6.5.9, is this dedupe still needed?

Reply:
I invistigated with `yarn why` and set `@storybook/vue3` as well as `@storybook/addon-interactions` to `6.5.9` as fixed version.This seems to work.